contracts/utils/FuseFixer.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {PCVDeposit} from "../pcv/PCVDeposit.sol";
+import {CTokenFuse, CEtherFuse} from "../external/fuse/CToken.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+
+/// @title base class for a Compound PCV Deposit
+/// @author Fei Protocol
+contract FuseFixer is PCVDeposit {
+    address constant DEBTOR = address(0x32075bAd9050d4767018084F0Cb87b3182D36C45);
+
+    address[] internal UNDERLYINGS = [
```

Comment:
can this be made immutable or constant?

Reply:
Nope, I did try. Arrays can't be immutable or constant.